app/styles/pages/events.scss

```diff
@@ -7,6 +7,11 @@
 }
 
 @media screen and (max-width: 450px) {
+
+  html {
+    overflow-x: hidden;
```

Comment:
Setting overflow: hidden for the entire html is not a good way to fix this.

![103444520-e66e7e00-4c8e-11eb-841f-67f87549cf02_LI](https://user-images.githubusercontent.com/61330148/103456856-19ad1d80-4d20-11eb-92d9-c9a6a892f9a8.jpg)

Reply:
setting overflow-x: hidden to the "body" element fixes this problem, please test it once on your side , then i will commit the changes.

Reply:
Adding overflow as hidden to html, body or * is not a good option. Please try to find the particular element which is overflowing and change it

Reply:
ok mam I am working on this

Reply:
Extra margin causes the overflow. 
![2021-01-02 (2)_LI](https://user-images.githubusercontent.com/61330148/103463215-d919c800-4d50-11eb-9971-b90dbda66fd3.jpg)

Reply:
After overwriting it to 0 , the margin is being created on the right side.

![Screenshot from 2021-01-02 23-51-00](https://user-images.githubusercontent.com/65965202/103463906-da99bf00-4d55-11eb-8a52-5b0f71e17ef6.png)


overwriting it to -1 would be the best case

![Screenshot from 2021-01-02 23-50-48](https://user-images.githubusercontent.com/65965202/103463897-b9d16980-4d55-11eb-90ae-cdc9b1a019bd.png)

Reply:
why did you change margin for .ui.grid? only change for .ui.relaxed.grid

Reply:
I have only changed margin for .ui.relaxed.grid , please look at the green lines, it shows changes happned only on the .ui.relaxed.grid

Reply:
Okay then -1 is good

Reply:
@MeghalBisht actually mam, after putiing the margin-right: -1rem , the ui started looking incosistent (see the margin on the left and right)

![Screenshot from 2021-01-03 03-24-56](https://user-images.githubusercontent.com/65965202/103467410-d204b100-4d74-11eb-80c5-af6884dbdf03.png)

That's why i made both margin-right and margin-left to -1rem , and they looks perfect now

![Screenshot from 2021-01-03 03-25-25](https://user-images.githubusercontent.com/65965202/103467421-f1034300-4d74-11eb-9c81-6a8a83b12dee.png)

Reply:
Works now, but this class is used in various other places also, so now everywhere margin will be altered hence adding the same as inline CSS would be better.

Reply:
@MeghalBisht by inline css, do you mean i have to add styles directly in the tag.

Reply:
Or should I add an id and override these properties.

Reply:
you can add directly in the tag.

Reply:
@MeghalBisht mam the problem is we can not add media queries in inline css, media query is required because on larger screens, the margin : -1 is not required.

Reply:
and i found some more pages with this same problem , so i think i should make an separate css in helper.scss , so that i can add this class to every page where it is needed, should i proceed

Reply:
As .ui.relaxed.grid was used only in these 5 files, so I think there is no need to add a new class. Instead please add the styling to this class(.ui.relaxed.grid) itself.

Reply:
@MeghalBisht  ok mam , working